packages/ag-charts-enterprise/src/components/color-picker/colorPickerStyles.css

```diff
@@ -60,7 +70,16 @@
         var(--ag-charts-focus-border-shadow);
 }
 
+.ag-charts-color-picker__color-row {
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    margin-bottom: 4px;
+    --inset: calc((var(--thumb-size) - var(--track-height)) / 2);
+}
+
 .ag-charts-color-picker__hue-input,
+.ag-charts-color-picker__multi-color-button,
```

Comment:
I think you probably don't want this selector here - this is very specific to the sliders. If anything breaks, copy the styles into `ag-charts-color-picker__multi-color-button`